player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -46,7 +46,7 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
     };
    
     $scope.replayContent = function() {
-        if(isMobile && ($rootScope.users.length > 1)) {
+        if(window.cordova && ($rootScope.users.length > 1)) {
```

Comment:
Relay on only isBrowserPreview flag always. Don't depend on window.corodva.

Reply:
As we are providing Cordova object to the player, window.cordova is available here, so it's working fine.